pkg/action/install.go

```diff
@@ -256,7 +257,12 @@ func (i *Install) RunWithContext(ctx context.Context, chrt *chart.Chart, vals ma
 		return nil, err
 	}
 
-	rel := i.createRelease(chrt, vals)
+	fmt.Println(driver.ContainsSystemLabels(i.Labels))
```

Comment:
Would we be able to remove this debugging line?

Reply:
```suggestion
	fmt.Println(driver.ContainsSystemLabels(i.Labels))
```

Reply:
Completely forgot about it, removed it. TY